.github/workflows/changelog.yml

```diff
@@ -21,10 +21,10 @@ jobs:
     steps:
       - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # https://github.com/actions/checkout/releases/tag/v2.3.4
 
-      - name: Use Node.js 12
+      - name: Use Node.js 15
```

Comment:
As the monorepo requires Node 15, can we add to the project root `.nvmrc` file with `v15` content? This can be helpful for developers working with Handsontable.

Reply:
great idea 👌